must_gather.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source logging.sh
+source common.sh
+
+MUST_GATHER_PATH=${MUST_GATHER_PATH:-$LOGDIR/must-gather}
```

Comment:
maybe the cluster name should be in this path, too?  For me this is just `logs/must-gather/`.

Reply:
Good point.  It probably makes sense to do that for all logs, right? I changed logging.sh to use the cluster name

Reply:
Actually, that's problematic, since this is always sources before common.sh.